code-challenges.test.js

```diff
@@ -10,43 +10,143 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
+// detailed pseudo code with proper vocabulary terms []
+// code that will output correct content in the terminal following best practices []
+// "your answers" written in complete sentences the student's own words that provide correct answers []
+// well researched and detailed answers written in complete sentences in the student's own words that provides additional context []
+
 // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
 
-// a) Create a test with expect statements for each of the variables provided.
+// Pseudo code:
 
+// < -- Sarah Proctor worked in class -- >
+
+// a) Create a test with expect statements for each of the variables provided.
 const numbersArray1 = [6, 7, 8, 9, 10]
 // Expected output: [18, 21, 24, 27, 30]
 const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
 
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
+// < -- Sarah Proctor worked in class -- >
 
-// --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
-// a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+// --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
+// Pseudo code:
+// 1. if / else statement
+// 2. is number % 3 === 0
+// 3. yes return num 'is divisible by three'
+// 4. else if return num 'is not divisible by three'
+
+const isDivisibleByThree = (number) => {
+    if (number % 3 === 0){
+        return `${number} is divisible by three`
+    } else if (number % 3 === 0) {
+        return `${number} is not divisible by three`
+    }
+}
+
+// console.log(isDivisibleByThree)
+
+
+// --------------------2) Create a test with expect statements for each of the variables provided.
+// Psuedo: 
+// 1. describe("object1",() =>
+// 2. it("a function that takes an object as an argument and decides if the number inside it is evenly divisible by three or not.", () =>
+// 3. { expect (object1(15) ).toEqual(TRUE)}
```

Comment:
The purpose of pseudo code is to plan your process in regular words not code

Reply:
Pseudocode is a detailed yet readable description of what a computer program or algorithm should do (according to google), but I understand the interest in wanting to understand someone else's notes. I'd be happy to talk you through my thought process on the work.